### PR TITLE
Created a sample code template for Keras to use exported models and u…

### PR DESCRIPTION
**Pull request by sotanishy**
…pdated the documentation

IMPORTANT NOTES (please read and then delete):

* The PR title should start with "Fix #bugnum: " (if applicable), followed by a clear one-line present-tense summary of the changes introduced in the PR. For example: "Fix #bugnum: Changed UI layout from horizontal to vertical.".

* Please make sure to mention "#bugnum" somewhere in the description of the PR. This enables Github to link the PR to the corresponding bug.

Please also make sure to follow the [contribution](https://github.com/Cloud-CV/Fabrik/blob/master/.github/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: example/train.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# Get the command line arguments
+model_file_name = ''
+try:
+    model_file_name = sys.argv[1]
+except IndexError:
+    print('Usage: python train.py model_json_file')
+    exit()
+
+# Load the dataset (keras.datasets.cifar10)
```
Comment (thatbrguy) [inline example/train.py]: You can add a comment on how users can upload their own datasets, or use other datasets from `keras.datasets`

File: tutorials/keras_json_usage_1.md [hunk 2]
```diff
@@ -73,3 +73,5 @@ You should be getting around 76-78% accuracy.
 
 This code trains and evaluates the loaded model on the dataset.
 
+### Code template
+[train.py](../example/train.py)
```
Comment (thatbrguy) [inline tutorials/keras_json_usage_1.md]: Elaborate on what the code template does and the dataset it uses. Most importantly, write a one line sample usage code. Something like: `python train.py MODEL_NAME`. Do mention that the user has to type in the model name. Also, you can rename `train.py` to something like `keras_sample_cifar10.py`

File: example/train.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+(x_train, y_train), (x_test, y_test) = cifar10.load_data()
+
+# Load the model from JSON file
+json_file = open(model_file_name, 'r')
+loaded_model_json = json_file.read()
+json_file.close()
+loaded_model = model_from_json(loaded_model_json)
+
```
Comment (thatbrguy) [inline example/train.py]:
```suggestion
loaded_model.summary()
```
You can print the model summary here using the above code. Add a one line comment to explain what this does. Optionally, you can get input from the user to either enable or disable printing model summary.

File: example/train.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+from keras.datasets import cifar10
```
Comment (Ram81) [inline example/train.py]: @sotanishy can you create a `code_samples` folder in `example/{framework}` based on the framework for which code is written. And put the samples there

**Comment by thatbrguy** [comment]
@sotanishy @Ram81 Looks good to me :+1:.

**Pull request merged**